src/core/src/toga/widgets/canvas.py

```diff
@@ -248,13 +236,14 @@ def bezier_curve_to(self, cp1x, cp1y, cp2x, cp2y, x, y):
 
         Returns:
             :class:`BezierCurveTo <BezierCurveTo>` object.
-
         """
         bezier_curve_to = BezierCurveTo(cp1x, cp1y, cp2x, cp2y, x, y)
         return self.add_draw_obj(bezier_curve_to)
 
     def quadratic_curve_to(self, cpx, cpy, x, y):
-        """Constructs and returns a :class:`QuadraticCurveTo <QuadraticCurveTo>`.
+        """Constructs and returns a :class:`QuadraticCurveTo.
+
+        <QuadraticCurveTo>`.
```

Comment:
As expected, docformatter broke the RST markup here, and there may be other places too.